### ISSUE TITLE
Rare failure of RandomizedRoutingTest for edge-based CH

### ISSUE THREAD

**Issue by easbar** [graphhopper/graphhopper#1916]
For this commit an CH (edge-based) test failed during a CI build: 79f3ecd606695f2510b67de06bcd28ed16affce9

```
[INFO] Running com.graphhopper.routing.weighting.RandomizedRoutingTest
expected: [50, 8, 35, 22]
given:    [50, 1, 22]
seed: 130339338847
[ERROR] Tests run: 60, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 5.595 s <<< FAILURE! - in com.graphhopper.routing.weighting.RandomizedRoutingTest
[ERROR] randomGraph_withQueryGraph[CH_ASTAR, EDGE_BASED](com.graphhopper.routing.weighting.RandomizedRoutingTest)  Time elapsed: 0.069 s  <<< FAILURE!
java.lang.AssertionError: 
wrong weight: 50->22
expected: 69.53626249436314
given:    71.0755881986408
seed: 130339338847
	at com.graphhopper.routing.weighting.RandomizedRoutingTest.comparePaths(RandomizedRoutingTest.java:353)
	at com.graphhopper.routing.weighting.RandomizedRoutingTest.runWithQueryGraph(RandomizedRoutingTest.java:310)
	at com.graphhopper.routing.weighting.RandomizedRoutingTest.randomGraph_withQueryGraph(RandomizedRoutingTest.java:282)
```

**Comment by easbar** [comment]
In this commit: dfa19a686c6efb44d5373a00059eff151a563704 some issues of this test were changed and now turn costs are also included. Also the encoder was changed from motorcycle to car. However the test still fails as of now (fb3eed814b567cfebde3bed20c3c273e7344c913) if we change the encoder back to Motorcycle (and use the above seed).

**Comment by easbar** [comment]
Here is another... d353297209ec871eca0468ce04f269739452beff

**Comment by easbar** [comment]
> However the test still fails as of now (fb3eed8) if we change the encoder back to Motorcycle (and use the above seed).

This is just a duplicate of #2061 

> Here is another... d353297

This is a rare case of a certain soft violation:

```
[
wrong distance 33->50, expected: 6745.514924478206, given: 7031.102924478206, wrong nodes 33->50
expected: [33, 44, 18, 39, 35, 16, 5, 23, 28, 43, 15, 0, 21, 2, 32, 50]
given:    [33, 44, 18, 39, 35, 17, 6, 5, 23, 28, 43, 15, 0, 21, 2, 32, 50], 
wrong distance 50->51, expected: 6800.420893766574, given: 7086.008893766575, wrong nodes 50->51
expected: [50, 44, 18, 39, 35, 16, 5, 23, 28, 43, 15, 34, 14, 7, 51]
given:    [50, 44, 18, 39, 35, 17, 6, 5, 23, 28, 43, 15, 34, 14, 7, 51]
]
```

Apparently the alternatives 35-16-5 and 35-17-6-5 have same weights, but different distances.

**Comment by easbar** [comment]
Fixed via #2215
